include/swift/AST/TypeRepr.h

```diff
@@ -465,60 +465,6 @@ inline IdentTypeRepr::ComponentRange IdentTypeRepr::getComponentRange() {
   return ComponentRange(this);
 }
 
-/// \brief A function type.
```

Comment:
Note to self: Moving this below TupleTypeRepr might be a mistake.  I should probably forward-declare and move the things accessing TupleTypeRepr into `TypeRepr.cpp`